src/browser/ui/dom/DOMPropertyOperations.js

```diff
@@ -15,8 +15,23 @@
 var DOMProperty = require('DOMProperty');
 
 var quoteAttributeValueForBrowser = require('quoteAttributeValueForBrowser');
+var invariant = require('invariant');
 var warning = require('warning');
 
+var VALID_ATTRIBUTENAME_REGEX = /^[a-zA-Z_][a-zA-Z_\.\-\d]*$/; // Simplified subset
```

Comment:
nit: `VALID_ATTRIBUTE_NAME_REGEX` (use a "space" since it's 2 words, just like you camelCased `validatedAttributeNameCache`)